product_docs/docs/jdbc_connector/42.5.4.2/installing/linux_ppc64le/index.mdx

```diff
@@ -11,7 +11,6 @@ redirects:
 
 navigation:
   - jdbc_rhel_9
-  - jdbc_rhel_8
```

Comment:
Why are we removing rhel_8?

Reply:
I'm not sure why that happened. I thought it was the templates do that, but that doesn't make any sense. I added it back in.